systemtest/src/test/java/io/strimzi/systemtest/kafka/QuotasST.java

```diff
@@ -37,19 +43,33 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.jupiter.api.Assumptions.assumeFalse;
 
-/**
- * NOTE: STs in this class will not properly work on `minikube` clusters (and maybe not on other clusters that uses local
- * storage), because the calculation of currently used storage is based
- * on the local storage, which can be shared across multiple Docker containers.
- * To properly run this suite, you should use cluster with proper storage.
- */
+@SuiteDoc(
+    description = @Desc("NOTE: STs in this class will not work properly on `minikube` clusters (and maybe not on other clusters that use local storage), because the calculation of currently used storage is based on the local storage, which can be shared across multiple Docker containers. To properly run this suite, you should use cluster with proper storage."),
+    beforeTestSteps = {
+        @Step(value = "Deploy default cluster operator with the required configurations.", expected = "Cluster operator is deployed.")
+    },
+    labels = {
+        @Label(value = TestDocsLabels.KAFKA)
+    }
+)
 public class QuotasST extends AbstractST {
     private static final Logger LOGGER = LogManager.getLogger(QuotasST.class);
 
-    /**
-     * Test to check Kafka Quotas Plugin for disk space
-     */
     @ParallelNamespaceTest
+    @TestDoc(
+        description = @Desc("Test to check Kafka Quotas Plugin for disk space."),
+        steps = {
+            @Step(value = "Assume the cluster is not Minikube or MicroShift.", expected = "Cluster is appropriate for the test."),
+            @Step(value = "Create necessary resources for Kafka and nodes.", expected = "Resources are created and Kafka is set up with quotas plugin."),
```

Comment:
IMHO this step would be better to describe more -> that you are configuring some quota for producer/consumer and adding some excluded principals, that should be excluded from the quota itself.

Reply:
Sure updated